### DISCUSSION TITLE
Support API reads from read-only / read-replica

### DISCUSSION THREAD

**Discussion by goggi** [directus/directus#3723] [Feature Requests]
Would be great if it was possible to connect Directus to a read-only / read-replica db and do API get calls from it. Currently, it seems the API calls are updating some tables (directus_users). Maybe just have a parameter that skips any database updates when doing gets from the API. 

Cheers!

**Comment by philleepflorence** [comment]
You should be able to add **Collections** to the [Public](https://docs.directus.io/concepts/users-roles-and-permissions.html) Role, if you don't want any **Authentication** - that might skip the `directus_users` collection.
Do you also wish to bypass [Activity](https://docs.directus.io/concepts/activity-and-versions.html)?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
